testhelper/destination/kafka.go

```diff
@@ -27,10 +27,20 @@ func SetupKafka(pool *dockertest.Pool, d deferer) (*KafkaResource, error) {
 		return nil, fmt.Errorf("Could not create docker network: %w", err)
 	}
 	d.Defer(func() error {
-		// TODO delete network
+		if err := pool.Client.RemoveNetwork(network.ID); err != nil {
+			return fmt.Errorf("Could not remove kafka network: %w \n", err)
+		}
 		return nil
 	})
 
+	networkInfo, err := pool.Client.NetworkInfo(network.ID)
+	if err != nil {
+		return nil, fmt.Errorf("Could not get docker network info: %w", err)
+	}
+	for _, ipamConfig := range networkInfo.IPAM.Config {
+		log.Println("kafka network subnet:", ipamConfig.Subnet)
+	}
```

Comment:
Do we need the log here?

Reply:
Need it, no. But there's still a chance we might get a subnet conflict with the clickhouse network, and this can help us understand if that is the case or not.

Reply:
@atzoum we're creating two docker networks now:
1. kafka_network: no particular settings, so will get an available subnet from docker service
2. clickhouse-network: requests the subnet `172.23.0.0/24` and fails if that subnet is not available, which is our current problem.

Ideally, we'd like to have no fixed subnets unless we really need them to. So, we want to discuss a bit with the warehouse devs about removing that if possible. Until that happens, this PR will mitigate our problem with failing CI tests.